deps/tools/common.mk

```diff
@@ -43,11 +43,6 @@ endif
 # NOTE: Do not make RPATH changes in CMAKE_COMMON on platforms other than FreeBSD, since
 # it will make its way into the LLVM build flags, and LLVM is picky about RPATH (though
 # apparently not on FreeBSD). Ref PR #22352
```

Comment:
I'd just put the comment block near the top of the file so that anyone who goes into the file wanting to make RPATH changes will find it.